src/tools/debugger/session/Breakpoints.java

```diff
@@ -74,6 +74,7 @@ public synchronized void addOrUpdate(final LineBreakpoint bId) {
       WebDebugger.log("LineBreakpoint: " + bId);
       bp = Breakpoint.newBuilder(bId.getURI()).
           lineIs(bId.getLine()).
+          steppingLocation(SteppingLocation.BEFORE_STATEMENT).
```

Comment:
which changes are you refering to? I only see this one as last commit https://github.com/smarr/truffle/commit/202248be40c6166a49ea664b8b2ef3e4e8980f96